docs/content/docs.md

```diff
@@ -571,6 +571,15 @@ This flag can be useful for debugging and in exceptional circumstances
 (e.g. Google Drive limiting the total volume of Server Side Copies to
 100GB/day).
 
+### --dscp VALUE ###
+
+Specify a DSCP value or name to use in connections. This could help QoS
+system to identify traffic class. EF, DF, CSx and AFxx are allowed.
+
+See the description of [differentiated services](https://en.wikipedia.org/wiki/Differentiated_services) to get an idea of
+this field. Setting this to 8 (CS1) to identify the flow to SCAVENGER class
+can avoid occupying too much bandwidth in a network with DiffServ support.
```

Comment:
I agree with @ncw asking for "some more docs". @Max-Sum It would be nice if you added a short practical example, like
```
You have service1 with QoS xxx, then you run e.g.
rclone copy --dscp yyy from:/from to:/to
and in result you get...
```

Reply:
I guess most rclone users copy bulk data about but want to lower the QoS on that so it doesn't interfere with their video conferencing/VOIP calls/Netflix viewing. What would be the best setting for that? Is that `CS1`?

I remember that @calebcase and I had a conversation about this because tardigrade uses this too - any thoughts?

Reply:
This is how we did it in storj/uplink: https://github.com/storj/common/tree/73bea234a20e88585742cfa6a794f10e7edfcaac/socket

We enabled it by default to help where we could with congestion issues.

Specifically: https://github.com/storj/common/blob/73bea234a20e88585742cfa6a794f10e7edfcaac/socket/background_linux.go#L19
Where `byte(dscpLE)<<2` (https://github.com/storj/common/blob/73bea234a20e88585742cfa6a794f10e7edfcaac/socket/dscp.go#L36) is `0b000001 << 2`
This was chosen based on the recommendations from https://tools.ietf.org/html/rfc8622#section-6 (which  recommends against `CS1`).

Reply:
How about I recommend people to set CS1 but not making it default? Because that might not be in people's favor.

Reply:
Thanks @calebcase - that RFC recommendation seems solid.

What do you think about making the recommended setting `LE` @Max-Sum ? Or perhaps we should say that the modern recommendation is `LE` but for older systems `CS1` might be appropriate.

Reply:
According to the RFC, CS1 might just not be used for this kind of flows. I think making LE the only recommanded setting would be appropriate.

Reply:
So `LE` stays as the only recommended setting.

@Max-Sum @ncw @calebcase 
I think we can consider this thread resolved.